packages/components/backtop/src/backtop.vue

```diff
@@ -28,69 +28,55 @@ import { backtopEmits, backtopProps } from './backtop'
 
 const COMPONENT_NAME = 'ElBacktop'
 
-export default defineComponent({
-  name: COMPONENT_NAME,
-  components: {
-    ElIcon,
-    CaretTop,
-  },
-  props: backtopProps,
-  emits: backtopEmits,
+defineOptions({
+  name: 'ElBacktop',
```

Comment:
@sxzz does `defineOptions` support non constant value?

Reply:
No, just like `defineProps`. It supports imported variables only.